flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/delegation/PlannerContext.java

```diff
@@ -83,8 +83,8 @@
 @Internal
 public class PlannerContext {
 
-    private final RelDataTypeSystem typeSystem = new FlinkTypeSystem();
-    private final FlinkTypeFactory typeFactory = new FlinkTypeFactory(typeSystem);
+    private final RelDataTypeSystem typeSystem;
```

Comment:
does this really need to change ?

Reply:
Oops，they should only appear in my another developing branch.